packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -1,12 +1,19 @@
 Terra.describeViewports('Clinical Item View', ['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], () => {
-  it('with one column displays', () => {
+  it('with displays', () => {
```

Comment:
I know this is following the previous sentence structure, but since we are updating these, we should reword these so it reads as a sentence starting with "it". E.g. "It renders with with displays" or any other more appropriate descriptor.

Reply:
sure thing, [cf417ee](https://github.com/cerner/terra-clinical/pull/912/commits/cf417ee24335fd1281d4babcd523b431e82945ea)